server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -162,9 +163,16 @@ export function create(
 				? uuid()
 				: (request.body.id as string) || uuid();
 
+			Lumberjack.info(
+				`Whole summary on First Summary ${request.body.enableWholeSummaryTreeOnFirstSummary}.`,
+			);
 			// Summary information
-			const summary = request.body.summary;
+			const summary =
+				request.body.enableWholeSummaryTreeOnFirstSummary === true
```

Comment:
You can do shorter version (i.e. remove === true):
request.body.enableWholeSummaryTreeOnFirstSummary ? caseA : caseB